doc/source/serve/production.md

```diff
@@ -567,3 +557,216 @@ This section summarizes the best practices when deploying to production:
+
+```console
+$ kubectl get rayservices
+NAME                AGE
+rayservice-sample   7s
+
+$ kubectl describe rayservice rayservice-sample
+```
+
+### Updating the application
+
+To update the `RayService`, modify the manifest and apply it use `kubectl apply`.
+There are two types of updates that can occur:
+- *Application-level updates*: when only the Serve config options are changed, the update will be applied _in-place_ on the same Ray cluster. This enables lightweight updates such as scaling a deployment up or down or modifying autoscaling parameters.
+- *Cluster-level updates*: when the `RayCluster` config options are changed, such as updating the container image for the cluster, it may result in a cluster-level update. In this case, a new cluster will be started, the application will be deployed to it, and once it is running the Kubernetes service will be updated to point at the new cluster. There should not be any downtime for the application.
+
+#### Example: Serve config update
+
+In the `FruitStand` example above, let's change the price of a mango in the Serve config to 4:
+
+```console
+  - name: MangoStand
+    numReplicas: 1
+    userConfig: |
+      price: 4
+```
+
+Now to update the application we apply the modified manifest:
+
+```console
+$ kubectl apply -f ray_v1alpha1_rayservice.yaml
+
+$ kubectl describe
+...
+  serveDeploymentStatuses:
+  - healthLastUpdateTime: "2022-07-18T21:51:37Z"
+    lastUpdateTime: "2022-07-18T21:51:41Z"
+    name: MangoStand
+    status: UPDATING
+...
+```
+
+If we query the application, we can see that we now get a different result reflecting the updated price:
+
+```console
+$ curl -X POST -H 'Content-Type: application/json' localhost:8000 -d '["MANGO", 2]'
+8
+```
+
+#### Updating the RayCluster config
+
+The process of updating the RayCluster config is the same as updating the Serve config.
+For example, we can update the number of worker nodes to 2 in the manifest:
+
+```console
+workerGroupSpecs:
+  # the pod replicas in this group typed worker
+  - replicas: 2
+```
+
+```console
+$ kubectl apply -f ray_v1alpha1_rayservice.yaml
+
+$ kubectl describe
+...
+  pendingServiceStatus:
+    appStatus: {}
+    dashboardStatus:
+      healthLastUpdateTime: "2022-07-18T21:54:53Z"
+      lastUpdateTime: "2022-07-18T21:54:54Z"
+    rayClusterName: rayservice-sample-raycluster-bshfr
+    rayClusterStatus: {}
+...
+```
+
+In the status, you can see that the `RayService` is preparing a pending cluster.
+After the pending cluster is healthy, it will be switched to the active cluster and the previous cluster will be terminated.
+
+## Failure Recovery
+
+Ray Serve is resilient to any component failures within the Ray cluster out of the box.
+You can read more about how process and worker node failures are handled in: {ref}`serve-ft-detail`.
```

Comment:
```suggestion
You can read more about how worker node and process failures are handled in: {ref}`serve-ft-detail`.
```

The other order makes it seem like there's two types of nodes called "process nodes" and "worker nodes."